all_repos/complete.py

```diff
@@ -46,6 +46,19 @@
 }
 '''
 
+ZSH = '''\
+__git_remote_repositories() {
+    local -a options
+    options=( $(
+        jq -r \
+           'to_entries | map([.value, .key]|join("\\t")) | join("\\n")' \
+           < "$__all_repos__repos_json" |
```

Comment:
do you need to pipe the file into `jq`? the bash one doesn't need it

Reply:
nope, good catch